docs/standards/standard-detection.md

```diff
@@ -60,29 +58,27 @@ await myUPContract.methods.supportsInterface(ERC725AccountInterfaceId).call();
 
 :::info
 
-See [ERC165 - Standard Interface Detection](https://eips.ethereum.org/EIPS/eip-165) for more infos.
+See [ERC165 - Standard Interface Detection](https://eips.ethereum.org/EIPS/eip-165) for more info.
 
 :::
 
 ## Metadata Detection
 
 :::success Tip
 
-All `SupportedStandards:{StandardName}` keys can be found under each ERC725Y JSON Schema on **[erc725.js](https://github.com/ERC725Alliance/erc725.js/tree/develop/src/schemas)**.
+The **[erc725.js](https://github.com/ERC725Alliance/erc725.js/tree/develop/src/schemas)** GitHub repository lists all the `SupportedStandards:{StandardName}` keys under each ERC725Y JSON Schema.
 
 :::
 
-> This covers how to detect if a contract contains a specific set of ERC725Y in its storage.
-
-We can verify if a contract contains a certain set of ERC725 keys (= **metadata**) by checking value stored under the key `SupportedStandards:{StandardName}` in the contract storage.
+> This section covers how to detect if a contract contains a specific set of ERC725Y in its storage.
 
-This can be done by calling the function `getData([SupportedStandards:{StandardName}])`, passing as a parameter the `bytes32` key of the Metadata Standard.
+We can verify if a contract contains a specific set of ERC725 keys (= **metadata**) by checking the value stored under the key `SupportedStandards:{StandardName}` in the contract storage, via the function `getData([SupportedStandards:{StandardName}])`.
 
-Calling this function will return a specific `bytes4` value (defined in the Metadata Standard) if the contract has some metadata keys set by default. Otherwise, it will return an empty value.
+Calling this function will return a specific bytes4 value (defined in the Metadata Standard) if the contract has some metadata keys set by default. Otherwise, it will return an empty value.
 
 ### Example
 
-An **[LSP7DigitalAsset](./nft-2.0/03-LSP7-Digital-Asset.md)** is a contract that contains ERC725Y keys defined in **[LSP4 - Digital Asset Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md)**. This means that the contract **SHOULD** have the following ERC725Y keys set by default: `LSP4TokenName`, `LSP4TokenSymbol`, `LSP4Metadata`, `LSP4CreatorsMap:<address>` and `LSP4Creators[]`.
+An **[LSP7DigitalAsset](./nft-2.0/03-LSP7-Digital-Asset.md)** is a contract that contains ERC725Y keys defined in **[LSP4 - Digital Asset Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md)**. Therefore, the contract **SHOULD** have the following ERC725Y keys set by default: `LSP4TokenName`, `LSP4TokenSymbol`, `LSP4Metadata`, `LSP4CreatorsMap:<address>` and `LSP4Creators[]`.
```

Comment:
We talk about the contract and link the standard of LSP7, if you want to link the standard and still talk about the contract, you could say:
An implementation of [LSP7-DigitalAsset](link to standard) is a contract that ...